src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -171,5 +176,13 @@ def load_command_table(self, _):
         g.custom_command('remove', 'application_configuration_service_git_remove')
         g.custom_command('list', 'application_configuration_service_git_list')
 
+    with self.command_group('spring-cloud build-service buildpacks-binding',
+                            custom_command_type=buildpacks_binding_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_command('create', 'buildpacks_binding_create')
```

Comment:
do you want to support no_wait?

Reply:
This is a question I want to know.
What's the difference between invoke request to RP with and without `sdk_no_wait`?

Reply:
https://github.com/Azure/azure-cli/blob/29767d75d850ddc1c24cc85bd46d861b61d77a47/doc/authoring_command_modules/authoring_commands.md#register-commands